Assets/Plugins/Source/Editor/Windows/DevPortalSettingsWindow.cs

```diff
@@ -22,17 +22,18 @@
 
 //#define ALLOW_CREATION_OF_EOS_CONFIG_AS_C_FILE
 
-namespace PlayEveryWare.EpicOnlineServices
+namespace PlayEveryWare.EpicOnlineServices.Editor.Windows
 {
     using System;
     using System.Collections.Generic;
     using System.IO;
     using UnityEditor;
     using UnityEngine;
+    using Utility;
     using Random = System.Random;
 
     [Serializable]
-    public class EpicOnlineServicesConfigEditor : EOSEditorWindow
+    public class DevPortalSettingsWindow : EOSEditorWindow
```

Comment:
I don't feel this is a reasonable name change because the settings here are grouped together not because they are from the dev portal, but because they are settings we need to bootstrap EOS that need to be available to both Managed code and Native code, including cases where the Native code is run before Unity is fully bootstrapped.

Reply:
That makes sense.

Reply:
It's been changed to a name that is more representative of the collection of configuration values being pertinent to the task of setting up the EOS system.